18F/identity-idp
# LG-7205: Log click on in-person troubleshooting option

**Why**: To have better insight into the user's journey through in-person proofing.

**Testing Instructions:**

1. Navigate to http://localhost:3000
2. Sign in
3. Navigate to http://localhost:3000/verify
4. Complete proofing flow up to document capture
5. Upload [attention result YAML test documents](https://developers.login.gov/testing/#data-testing)
6. Click Submit
7. Click "Verify your identity in person"
8. In your terminal, run `tail -n2 log/events.log`
9. Observe logged event for "IdV: verify in person troubleshooting option clicked"
